packages/picasso/src/SidebarMenu/SidebarMenu.tsx

```diff
@@ -35,12 +35,14 @@ export const SidebarMenu = forwardRef<HTMLUListElement, Props>(
     const {
       variant,
       expandedItemKey,
-      setExpandedItemKey
+      setExpandedItemKey,
+      isCollapsed: isSidebarCollapsed
     } = useContext<SidebarContextProps>(SidebarContext)
 
-    const expandSidebarItem = useCallback(index => setExpandedItemKey(index), [
-      setExpandedItemKey
-    ])
+    const expandSidebarItem = useCallback(
+      index => setExpandedItemKey(index),
+      [setExpandedItemKey]
+    )
```

Comment:
This could be simplified to just:
```suggestion
    const expandSidebarItem = useCallback(
      setExpandedItemKey,
      [setExpandedItemKey]
    )
```